src/components/Table/TableHeader.tsx

```diff
@@ -0,0 +1,19 @@
+import { ReactNode } from 'react'
+
+export interface TableProps {
+  className?: string
+  children: ReactNode
+}
+
+export function TableHeader (props: TableProps): JSX.Element {
+  const { className = '', children } = props
+  return (
+    <div className={`table-header flex ${className}`}>
```

Comment:
use `classNames`, see other component that uses it

Reply:
Deleted this component.